sysutils/osxbom/Portfile

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# My repo is ahead of the main one in terms of versions:
+github.setup        cooljeanius osxbom 0.0.5
+github.tarball_from archive
+categories          sysutils
+maintainers         {gwmail.gwu.edu:egall @cooljeanius} openmaintainer
+platforms           darwin
+license             public-domain
+
+description         A free re-implementation of the NextSTEP/OSX lsbom \
+                    utility
+long_description    ${description}. Used by the PureDarwin project \
+                    to work with MacPorts binary packages.
+homepage            https://cauldrondevelopment.com/svn/${name}/trunk/
+
+checksums           rmd160  ae9fa1120123aa0f19d1a92bce869d530556bf19 \
+                    sha256  788c6650494eff481baebe1129335003301d9530c20ad61cb0ec013b628797ca \
+                    size    155958
+
+# probably not actually necessary to autoreconf, but whatever:
+use_autoreconf      yes
+autoreconf.args     -fvi -Wall
+
+# mostly just things picked up by trace mode:
+depends_build-append \
+                    port:gawk \
+                    port:grep \
+                    bin:nroff:groff \
+                    bin:gmkdir:coreutils \
+                    port:libmacho-headers
+depends_lib-append  port:libunwind
+
+set CFLAGS          "${configure.cflags} [get_canonical_archflags cc]"
+
+build.env-append    CC=${configure.cc} \
+                    CXX=${configure.cxx} \
+                    CPP=${configure.cpp}
+
+pre-build {
+    reinplace "s|-Werror|${CFLAGS}|" ${worksrcpath}/Makefile
+}
+
+test.run            yes
+test.target         check
+test.target-append  distcheck
+depends_test-append bin:gzip:gzip \
+                    bin:zip:zip
+
+destroot {
+        xinstall ${worksrcpath}/lsbom ${destroot}${prefix}/bin
+        xinstall ${worksrcpath}/bom.h ${destroot}${prefix}/include
+}
+
+variant rename description {Stop osxbom from shadowing the system lsbom} {
+    destroot {
+        xinstall ${worksrcpath}/lsbom ${destroot}${prefix}/bin/${name}
+        xinstall -d ${destroot}${prefix}/include/${name}
+        xinstall ${worksrcpath}/bom.h ${destroot}${prefix}/include/${name}
+    }
+}
+
+default_variants    +rename
+
+variant universal {}
```

Comment:
If the project can't be compiled as a universal binary, simply use the `universal_variant` field and set it to `no`. Otherwise, this can be dropped.

Reply:
I'm not currently on a universal-capable system, so I can't really test if it works or not; I'll just assume I can drop it... removed in 6c41a84.